### ISSUE TITLE
Launching ubuntu creates blank screen

### ISSUE THREAD

**Issue by ghost** [dnschneid/crouton#3898]
```
Please paste the output of the following command here: sudo startunity
```
![image](https://user-images.githubusercontent.com/29547944/46158068-b976c980-c242-11e8-9f52-097bf4f3798f.png)

#### Please describe your issue:
I installed Ubuntu by doing      sudo sh -e ~/Downloads/crouton -t unity
and when i launched it it only showed a blank desktop

**Comment by DennisLfromGA** [comment]
@nmkrupp,

#### Please paste the output of the following command below: `sudo edit-chroot -all`

**Comment by dmlyons** [comment]
I am getting this problem as well, here is my `startunity` and `edit-chroot -all`:

```
chronos@localhost ~ $ sudo enter-chroot
Entering /mnt/stateful_partition/crouton/chroots/xenial...
Unknown username "geoclue" in message bus configuration file
(xenial)dl@localhost:~$ startunity 

_XSERVTransmkdir: Owner of /tmp/.X11-unix should be set to root

X.Org X Server 1.18.4
Release Date: 2016-07-19
X Protocol Version 11, Revision 0
Build Operating System: Linux 4.4.0-128-generic x86_64 Ubuntu
Current Operating System: Linux localhost 3.18.0-18543-gac154810530e #1 SMP PREEMPT Wed Sep 26 00:36:13 PDT 2018 x86_64
Kernel command line: cros_secure console= loglevel=7 init=/sbin/init cros_secure oops=panic panic=-1 root=/dev/dm-0 rootwait ro dm_verity.error_behavior=3 dm_verity.max_bios=-1 dm_verity.dev_wait=1 dm="1 vroot none ro 1,0 3584000 verity payload=PARTUUID=f2e60aad-4c24-cf43-b6cf-b4e68926e40f/PARTNROFF=1 hashtree=PARTUUID=f2e60aad-4c24-cf43-b6cf-b4e68926e40f/PARTNROFF=1 hashstart=3584000 alg=sha1 root_hexdigest=05ee24d781d171ef0f1fca1cf0be8dcdb7ca4988 salt=b31baadcd4fd5980bd7aa2dd4744d4a3c1ff462c6efc1787aa041b13b8753847" noinitrd vt.global_cursor_default=0 kern_guid=f2e60aad-4c24-cf43-b6cf-b4e68926e40f add_efi_memmap boot=local noresume noswap i915.modeset=1 tpm_tis.force=1 tpm_tis.interrupts=0 nmi_watchdog=panic,lapic intel_idle.max_cstate=7  
Build Date: 10 August 2018  09:33:05AM
xorg-server 2:1.18.4-0ubuntu0.8 (For technical support please see http://www.ubuntu.com/support) 
Current version of pixman: 0.33.6
        Before reporting problems, check http://wiki.x.org
        to make sure that you have the latest version.
Markers: (--) probed, (**) from config file, (==) default setting,
        (++) from command line, (!!) notice, (II) informational,
        (WW) warning, (EE) error, (NI) not implemented, (??) unknown.
(++) Log file: "/tmp/Xorg.crouton.1.log", Time: Sun Sep 30 17:57:36 2018
(==) Using system config directory "/usr/share/X11/xorg.conf.d"
Error org.freedesktop.DBus.Error.ServiceUnknown: The name org.chromium.LibCrosService was not provided by any .service files
xf86EnableIOPorts: failed to set IOPL for I/O (Operation not permitted)
Error executing command as another user: Not authorized

This incident has been reported.
crouton: version 1-20180718130616~master:5d4c3014
release: xenial
architecture: amd64
xmethod: xorg
targets: touch,unity-desktop
host: version 11021.28.0 (Official Build) beta-channel cave 
kernel: Linux localhost 3.18.0-18543-gac154810530e #1 SMP PREEMPT Wed Sep 26 00:36:13 PDT 2018 x86_64 x86_64 x86_64 GNU/Linux
freon: yes
xrandr: Output eDP1 is not disconnected but has no modes
xrandr: Output DP1 is not disconnected but has no modes
xrandr: Output DP2 is not disconnected but has no modes
xrandr: Output HDMI1 is not disconnected but has no modes
xrandr: Output HDMI2 is not disconnected but has no modes
Kicking xrandr again
xrandr: Output eDP1 is not disconnected but has no modes
xrandr: Output DP1 is not disconnected but has no modes
xrandr: Output DP2 is not disconnected but has no modes
xrandr: Output HDMI1 is not disconnected but has no modes
xrandr: Output HDMI2 is not disconnected but has no modes
Kicking xrandr again
(II) AIGLX: Suspending AIGLX clients for VT switch
Error org.freedesktop.DBus.Error.ServiceUnknown: The name org.chromium.LibCrosService was not provided by any .service files
^C/usr/bin/xinit: connection to X server lost

waiting for X server to shut down Hangup
Hangup
Running exit commands...
(II) Server terminated successfully (0). Closing log file.

/usr/bin/xinit: unexpected signal 2
(xenial)dl@localhost:~$ logout
Unmounting /mnt/stateful_partition/crouton/chroots/xenial...
chronos@localhost ~ $ sudo edit-chroot -all
name: xenial
encrypted: no
Entering /mnt/stateful_partition/crouton/chroots/xenial...
crouton: version 1-20180718130616~master:5d4c3014
release: xenial
architecture: amd64
xmethod: xorg
targets: touch,unity-desktop
host: version 11021.28.0 (Official Build) beta-channel cave 
kernel: Linux localhost 3.18.0-18543-gac154810530e #1 SMP PREEMPT Wed Sep 26 00:36:13 PDT 2018 x86_64 x86_64 x86_64 GNU/Linux
freon: yes
Unmounting /mnt/stateful_partition/crouton/chroots/xenial...
```

Any help would be appreciated.

**Comment by DennisLfromGA** [comment]
@nmkrupp,

Do you get a blank screen when launched from outside the chroot in a shell session with:

- `sudo startunity` 

You could try adding a different desktop/target like lxde-desktop instead of unity.
You could also try adding the 'xiwi' xmethod/target and see if unity works with it.

*Hope this helps,*
-**DennisLfromGA**

**Comment by saurabh76** [comment]
@DennisLfromGA  
I had the exact same issue - ie I was getting a blank desktop.

Note: I had been using **beta** channel on my chromebook from last June or so and crouton had been running perfectly, it was only when I reverted to **stable** channel from the beta, that I got this error.

Your tip above of adding xiwi method worked beatifully and resolved my issue!

For the benefit of others and @dmlyons @nmkrupp, here is the command (assuming you are using xenial i.e. Ubuntu 16.04)
`sudo sh ~/Downloads/crouton -t xiwi -u -n xenial`

PS: I am using an ASUS C302c Chromebook
